Личный проект: завершаем разметку

HunteX opened the pull request:
---
:mortar_board: [Личный проект: завершаем разметку ](https://up.htmlacademy.ru/htmlcss/27/user/1316157/tasks/10)

:boom: https://htmlacademy-htmlcss.github.io/1316157-nerds-27/7/

senchkim commented:
1) всплывающее окно нужно разместить ниже футера

HunteX commented:
> 1. всплывающее окно нужно разместить ниже футера

сделал

senchkim commented:
2) у социальных иконок нет текстового описания можно использовать класс `visually-hidden` либо же атрибут `aria-label`

senchkim commented:
3) ссылка в логотипе на второй странице должна вести на главную`<a href="index.html">`

senchkim commented:
4) `<li>По цене</li>`
      `<li>По типу</li>`
      `<li>По названию</li>`
вот это еще нужно положить в ссылку так как это фильтр и при клике должен меняться контент!

senchkim commented:
5) ну и в целом классов не хватает - а их придется добавлять потому что стилизация у нас по классам!

HunteX commented:
> 1. у социальных иконок нет текстового описания можно использовать класс `visually-hidden` либо же атрибут `aria-label` - исправить на обоих страницах

fixed

> 1. ссылка в логотипе на второй странице должна вести на главную`<a href="index.html">`

fixed

> 1. `<li>По цене</li>`
>    `<li>По типу</li>`
>    `<li>По названию</li>`
>    вот это еще нужно положить в ссылку так как это фильтр и при клике должен меняться контент!

fixed

> 1. ну и в целом классов не хватает - а их придется добавлять потому что стилизация у нас по классам!

понял, в рамках этого задания пока не добавляю классы, планирую это сделать в следующем
